app.py

```diff
@@ -28,25 +28,26 @@ def home():
   return 'Welcome to the Islam Buddy API!'
 
 
-"""
```

Comment:
Just to confirm, this is still a debug endpoint but we no longer add using the add command correct? If so, when we push to prod, lets not expose it anyways?

Reply:
yeah i'm still figuring out a good way to manage this, but for now it's helpful for debugging